### PR TITLE
feat: add strchr and strrchr

### PR DESCRIPTION
**Pull request by keyboard-slayer**

### CODE REVIEW

File: src/string/strchr.c [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    while (*s)
+    {
+        if (*s == c)
+        {
+            return ((char *) s);
+        }
+        s++;
+    }
+
+    return (0);
```
Comment (Supercip971) [inline src/string/strchr.c:13]: If we want to be 100% standard compliant, we should use the NULL macro, as it is not 100% guarantee that it is (void*)0.

From the c standard: 
> The macros are
> NULL
> which expands to an **implementation-defined** null pointer constant;

File: src/string/strrchr.c [hunk 2]
```diff
@@ -0,0 +1,16 @@
+char *
+strrchr(char const *s, int c)
+{
+    char *last = 0;
+
+    while (*s)
+    {
+        if (*s == c)
+        {
+            last = (char *) s;
```
Comment (d0p1s4m4) [inline src/string/strrchr.c:10]: pas d'espace entre ) et s

File: src/string/strchr.c [hunk 3]
```diff
@@ -0,0 +1,14 @@
+char *
+strchr(const char *s, int c)
+{
+    while (*s)
+    {
+        if (*s == c)
+        {
+            return ((char *) s);
```
Comment (d0p1s4m4) [inline src/string/strchr.c:8]: pas d'espace

File: src/string/strchr.c [hunk 4]
```diff
@@ -0,0 +1,14 @@
+char *
+strchr(const char *s, int c)
+{
+    while (*s)
```
Comment (d0p1s4m4) [inline src/string/strchr.c:4]: pas d'espace entre `while` et `(`

File: src/string/strchr.c (hunk 1 above)
Comment (d0p1s4m4) [reply src/string/strchr.c:13]: `return (NULL)` instead of 0

**Review by d0p1s4m4 — CHANGES_REQUESTED** [review]

**Review by d0p1s4m4 — APPROVED** [review]

**Pull request merged**